pandas-stubs/core/frame.pyi

```diff
@@ -150,28 +151,25 @@ class _LocIndexerFrame(_LocIndexer):
     @overload
     def __getitem__(
         self,
-        idx: tuple[int | StrLike | tuple[HashableT, ...], int | StrLike],
+        idx: tuple[int | StrLike | tuple[ScalarT, ...], int | StrLike],
```

Comment:
`HashableT` included `slice` and if it is a `slice` in the tuple, we are not getting a scalar with `.loc`